src/main.rs

```diff
@@ -235,6 +236,13 @@ fn config_options<'a, 'b>() -> App<'a, 'b> {
                     Arg::with_name("no_iat")
                         .help("prevent an iat claim from being automatically added")
                         .long("no-iat")
+                ).arg(
+                    Arg::with_name("base64")
```

Comment:
I don't think the flag should be `--base64`. Primarily because that feels like a flag for ALL options, so the secret, JSON, payloads, etc., would then be interpreted as base64. What do you think of a longer `--secret-base64`, `--secret-b64`, `--secretb64`, or `--secret64`?

Reply:
I'm happy to do whatever you prefer. To me, `--secret-base64` (or those other variations) feels more like an option than a flag; e.g.
```
jwt encode --secret-base64 <base64 here> ...
```
This was actually my first thought, but having two ways of providing the secret means that neither of them can be strictly required. There would need to be validation logic somewhere to ensure that at least one of them was specified.

I'm fine with using a longer flag name too. If we go with a longer flag name, perhaps we do want a shortcut for it?

What's your preference?

Reply:
Just looking at the docs for Clap, since mutual exclusivity takes precedence over "required by default", I guess we could do `--secret-base64` as an option by configuring it such that it is not required, and configuring `--secret` as required and `conflicts_with("secret-base64")`. Just a thought...

Reply:
Yeah, this is tough to find a good solution for. Do you have examples of other CLI projects that have something like this? Another option would be to have a `--secret-format` flag, but that opens up this PR to adding support for file beyond `@`.

That being said, maybe we can add a formatting option to the secret? We have `--secret=@path/to/file`, so maybe something like `--secret=base64;secretHere`?

Reply:
I kinda like the idea of specifying the format as part of the `--secret` option because it would open up the possibility of supporting other common encodings if that was ever desired.

```
--secret b64:<base64 data>
--secret hex:<hexadecimal data>
--secret str:<secret string>          # useful if secret happens to start with `@` or `b64:`, etc
```